app/src/main/java/org/sopt/official/feature/main/MainViewModel.kt

```diff
@@ -1,25 +1,184 @@
+            .stateInLazily(viewModelScope)
+    val profileImage: StateFlow<NullableWrapper<String>>
+        get() = mainViewResult
+            .map { it.get()?.user?.profileImage?.get().asNullableWrapper() }
+            .stateInLazily(viewModelScope)
+    val generationList: StateFlow<NullableWrapper<List<Long>>>
+        get() = mainViewResult
+            .map { it.get()?.user?.generationList?.get().asNullableWrapper() }
+            .stateInLazily(viewModelScope)
+    val attendanceScore: StateFlow<NullableWrapper<Double>>
+        get() = mainViewResult
+            .map { it.get()?.operation?.attendanceScore?.get().asNullableWrapper() }
+            .stateInLazily(viewModelScope)
+    val announcement: StateFlow<NullableWrapper<String>>
+        get() = mainViewResult
+            .map { it.get()?.operation?.announcement?.get().asNullableWrapper() }
+            .stateInLazily(viewModelScope)
+    val blockItem: StateFlow<NullableWrapper<Triple<LargeBlockType, SmallBlockType, SmallBlockType>>>
+        get() = mainViewResult
+            .map {
+                val userState = it.get()?.user?.status ?: UserState.UNAUTHENTICATED
+                when (userState) {
+                    UserState.ACTIVE -> Triple(
+                        LargeBlockType.SOPT_OFFICIAL_PAGE_URL,
+                        SmallBlockType.SOPT_REVIEW_URL,
+                        SmallBlockType.SOPT_PROJECT_URL
+                    )
+                    UserState.INACTIVE -> Triple(
+                        LargeBlockType.SOPT_ATTENDENCE,
+                        SmallBlockType.PLAYGROUNG_MEMBER_URL,
+                        SmallBlockType.PLAYGROUNG_PROJECT_URL
+                    )
+                    UserState.UNAUTHENTICATED -> Triple(
+                        LargeBlockType.SOPT_FAQ_URL,
+                        SmallBlockType.PLAYGROUNG_MEMBER_URL,
+                        SmallBlockType.PLAYGROUNG_PROJECT_URL
+                    )
+                }.asNullableWrapper()
+            }
+            .stateInLazily(viewModelScope)
+    val blockList: StateFlow<List<SmallBlockItemHolder>>
+        get() = mainViewResult
+            .map {
+                val userState = it.get()?.user?.status ?: UserState.UNAUTHENTICATED
+                when (userState) {
+                    UserState.ACTIVE -> listOf(SmallBlockType.SOPT_FAQ_URL, SmallBlockType.SOPT_OFFICIAL_YOUTUBE)
+                    UserState.INACTIVE -> listOf(SmallBlockType.SOPT_OFFICIAL_PAGE_URL, SmallBlockType.PLAYGROUNG_CREW_URL)
+                    UserState.UNAUTHENTICATED -> listOf(SmallBlockType.PLAYGROUNG_CREW_URL, SmallBlockType.SOPT_OFFICIAL_PAGE_URL)
+                }
+            }
+            .map { list -> list.map { SmallBlockItemHolder.SmallBlock(it) } }
+            .stateInLazily(viewModelScope, emptyList())
+
+    init {
+        getMainView()
+    }
 
-    sealed class SmallBlockItemHolder: BaseItemType {
+    fun getMainView() {
```

Comment:
```suggestion
    fun initMainView() {
```
뷰 초기화라는 의미에서 init은 어떰?